integrations/comscore/lib/index.js

```diff
@@ -89,6 +92,23 @@ Comscore.prototype.mapComscoreParams = function(page) {
     }
   });
 
+  if (this.options.consentFlag) {
+    consentValue = page.proxy('properties.' + this.options.consentFlag);
+    if (this.options.consentFlag in properties) {
+      if (String(consentValue) === 'true' || String(consentValue) === '1') {
+        consentValue = '1';
+      } else if (
+        String(consentValue) === 'false' ||
+        String(consentValue) === '0'
+      ) {
```

Comment:
@longstoryshort super small change, but can we make sure the formatting is standardized? Looks like the first `if` statements are on a single line, but this one for `false` / `0` values is on separate lines.

Reply:
@kdaswani Formatting updated, please see [here](https://github.com/segmentio/analytics.js-integrations/pull/623/commits/14a0e7df884200a35d0c761d3791c21b57d8b7be).